### DISCUSSION TITLE
What do you think to bring back a C# Multiplatform gui to alive ? (I am speaking about Wx.Net)

### DISCUSSION THREAD

**Discussion by gnubsdguy** [Nitrux/nitrux-bug-tracker#140] [Ideas]
Hi, Hello, I have been practising and learning Mono C# and using Monodevelop as my main IDE and develop platform for C# On Linux(I am learning C++ for knowledge). But what I cannot find is a standard GUI for Mono or Mono Develop (there is not a complete Win forms, and we only have GTK# (which I like it) I have been looking for something else and there was something called as WX.Net (check on source forge about this project). I know that all of you keep busy with maintenance of the project mono, monodevelop and doing so many other things. I was thinking about because Wx Widgets is very well-used on Windows, Mac and Linux to bring alive again this project (I mean WX.NET it has not being updated since 2010) and may be to have another standard GUI library for multiplatform development with C# and Mono. There are so many people on different forums asking what happened to WX.NET. I don't know but it would be nice to know what the developers of the mono project will think about this.

**Comment by UriHerrera** [comment]
Sorry, I'm not sure what you're talking about. We do not develop, maintain, or use software that uses Mono, C#, or the .NET Framework.
